opennms-base-assembly/pom.xml

```diff
@@ -93,6 +93,10 @@
           </execution>
         </executions>
       </plugin>
+      <!--
```

Comment:
While running "assemble.pl" I stumbled upon the following log lines:

```
INFO] Compiling report file: subreports/ResponseTimeSummary_subreport.jrxml
[INFO] Compiling report file: InterfaceAvailabilityReport.jrxml
[INFO] Compiling report file: ResponseTimeCharts.jrxml
[INFO] Compiling report file: DiskUsageForCTX.jrxml
[INFO] Compiling report file: subreports/PeakTraffic_subreport.jrxml
[INFO] Compiling report file: subreports/Top25PercentDown_subreport.jrxml
[INFO] Compiling report file: ResponseTimeSummary.jrxml
[INFO] Compiling report file: AvailabilitySummary.jrxml
[INFO] Compiling report file: subreports/ResponseTimeSummary_Response_Offenders_subreport.jrxml
[INFO] Compiling report file: subreports/DiskUsageForCTXServers_subreport1.jrxml
[INFO] Compiling report file: subreports/ResponseTimeSummary_Availability_subreport.jrxml
[INFO] Compiling report file: Early-Morning-Report.jrxml
[INFO] Compiling report file: subreports/NodeId_to_NodeLabel_subreport.jrxml
[INFO] Compiling report file: NodeAvailabilityReport.jrxml
[INFO] Compiling report file: subreports/ResponseTime_subreport1.jrxml
[INFO] Compiling report file: subreports/AvailabilitySummaryChart_subreport.jrxml
[INFO] Compiling report file: subreports/ResponseTimeSummary_Availability_Offenders_subreport.jrxml
[INFO] Compiling report file: sample-report.jrxml
[INFO] Compiling report file: ResponseTime.jrxml
[INFO] Compiling report file: AssetManagementMaintStrategy.jrxml
[INFO] Compiling report file: subreports/95thPercentileTrafficRate_subreport.jrxml
[INFO] Compiling report file: SerialInterfaceUtilizationSummary.jrxml
[INFO] Compiling report file: TotalBytesTransferredByInterface.jrxml
[INFO] Compiling report file: AssetManagementMaintExpired.jrxml
[INFO] Compiling report file: subreports/InterfaceAvailabilityReport_subreport1.jrxml
[INFO] Compiling report file: TopIOWait.jrxml
[INFO] Compiling report file: AveragePeakTrafficRates.jrxml
[INFO] Compiling report file: subreports/TotalBytesTransferredByInterface_subreport1.jrxml
[INFO] Compiling report file: SnmpInterfaceOperAvailabilityReport.jrxml
[INFO] Compiled 30 report design files.
```

It seems jasper reports are still compiled